packages/flutter_tools/templates/app/ios.tmpl/Flutter/AppFrameworkInfo.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
   <key>CFBundleDevelopmentRegion</key>
-  <string>$(DEVELOPMENT_LANGUAGE)</string>
+  <string>en</string>
```

Comment:
out of curiosity, what does this key do? what are the consequences of using `$(DEVELOPMENT_LANGUAGE)` unprocessed?

Reply:
I think it's used as a fallback if the user's preferred language localization resources (strings, images, etc) aren't available, and `en` is the default.  App.framework doesn't contain any localized resources though.